charts/group-stack/values.yaml.Example

```diff
@@ -0,0 +1,72 @@
+fullnameOverride: "groupsmanager"
```

Comment:
this file doesn't have sensitive data... remove .Example from the file name.. its not needed and will remove the need to have a DevValues file if you are using the defaults